### ISSUE TITLE
Can't install vue3-openlayers 6.0.1 due to dependencies errors

### ISSUE THREAD

**Issue by lmc3s3f0r** [MelihAltintas/vue3-openlayers#308]
**Describe the bug**
Tried to upgrade to vue3-openlayers 6.0.1, so I added these dependencies to my project's package.json:

```
"ol-contextmenu": "^5.3.0",
"ol-ext": "^4.0.15",
"ol": "^9.0.0",
"vue": "3.4.21",
"vue3-openlayers": "^6.0.1"
```

And when I run npm install, I instantly get these error messages:
```
npm ERR! While resolving: nemus-dashboard@0.13.0
npm ERR! Found: ol@9.0.0
npm ERR! node_modules/ol
npm ERR!   ol@"^9.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer ol@"> 7.x <= 8.x" from ol-contextmenu@5.3.0
npm ERR! node_modules/ol-contextmenu
npm ERR!   ol-contextmenu@"^5.3.0" from the root project
```

Which I can confirm in ol-contextmenu's package.json: https://github.com/jonataswalker/ol-contextmenu/blob/master/package.json

```
 "peerDependencies": {
        "ol": "> 7.x <= 8.x"
    },
```

**Affected version(s)**
6.0.1

**To Reproduce**

1. Add this to your project's package.json:
```
"ol-contextmenu": "^5.3.0",
"ol-ext": "^4.0.15",
"ol": "^9.0.0",
"vue": "3.4.21",
"vue3-openlayers": "^6.0.1"
```
2. Delete node_modules and package-lock.json
3. Run npm intall

**Expected behavior**
I'd expect being able to install without issues.

**Desktop (please complete the following information):**

- OS: Ubuntu 22
- Browser: not relevant (it's in VSCode)

**Comment by d-koppenhagen** [comment]
Hey, please check out the issue [here](https://github.com/jonataswalker/ol-contextmenu/issues/277) with the workaround for your `package.json`:

```json
"overrides": {
    "ol-contextmenu": {
      "ol": "^9.x"
    }
  }
```

`ol-contextmenu` will be updated hopefully soon. The override should be fine for now, since there is no breaking change from OpenLayers 9 which breaks `ol-contextmenu`.

**Comment by d-koppenhagen** [comment]
I created a PR: https://github.com/jonataswalker/ol-contextmenu/pull/278

**Comment by d-koppenhagen** [comment]
fixed with version 6.1.1
